Core/src/Geometry/Layer.cpp

```diff
@@ -180,7 +180,7 @@ Acts::Layer::compatibleSurfaces(
     if (sfi && detail::checkIntersection(sfi.intersection, pathLimit,
                                          overstepLimit, s_onSurfaceTolerance)) {
       // Now put the right sign on it
-      sfi.intersection.pathLength *= std::copysign(1., options.navDir);
+      sfi.intersection.pathLength *= options.navDir;
```

Comment:
Doesn't this need to be explicitly converted to an integer (or underlying type) in this case?

Reply:
I defined the operators for these types.